dpnp/dpnp_algo/dpnp_algo_mathematical.pyx

```diff
@@ -139,54 +141,28 @@ cpdef dparray dpnp_cross(dparray x1, dparray x2):
     return call_fptr_2in_1out(DPNP_FN_CROSS, x1, x2, x1.shape)
 
 
-cpdef dparray dpnp_cumprod(dparray x1, bint usenan=False):
+cpdef dparray dpnp_cumprod(dparray x1):
+    #instead of x1.shape, (x1.size, ) is passed to the function
```

Comment:
Please, make this comment section as a docstring, if it is needed or just remove it.

Reply:
It looks like a comment for developers (internal comment) for better understanding the code.

Reply:
This no make sense, this comment includes example. Currently this comments are not so informative so this is needed to rewrite it.